framework/CodeInterfaces/Dymola/DymolaInterface.py

```diff
@@ -117,6 +117,22 @@ def __init__(self):
       @ In, None
       @ Out, None
     """
+    self.variablesToLoad = []
+
+  def _readMoreXML(self,xmlNode):
+    """
+      Function to read the portion of the xml input that belongs to this specialized class and initialize
+      some members based on inputs. This can be overloaded in specialize code interface in order to
+      read specific flags.
+      Only one option is possible. You can choose here, if multi-deck mode is activated, from which deck you want to load the results
+      @ In, xmlNode, xml.etree.ElementTree.Element, Xml element node
+      @ Out, None.
+    """
+    for child in xmlNode:
+      if child.tag == 'outputVariablesToLoad':
+        self.variablesToLoad = [var.strip() for var in child.text.split(",")]
```

Comment:
Modelica allows for variables that contain commas.
We can just specify that we don't support that for the moment?

Reply:
I can change the "comma separated list" into "space separated list".

Reply:
I changed it into "space separated list"... Now also the variables with commas can be handled.

Reply:
That would be good. I will wait with merging until you commit the change. Thanks